content/applications/sales/crm/optimize/partner_autocomplete.rst

```diff
@@ -1,48 +1,82 @@
 
-Enrich your contacts with corporate data
-========================================
+Enrich contacts with corporate data
+===================================
 
-| From any module, once you start to create a new contact typing the name of it, the system
-  will suggest a potential match. If you select it, the contact will be populated with corporate
-  data.
-| For example, after typing *Odoo*, you will get the following information:
+From any module, as the user is typing in the name of a new company contact, Odoo reveals a large
+drop-down menu of potential match suggestions. If any are selected, the contact is then populated
+with corporate data related to that specific selection.
 
-.. image:: partner_autocomplete/odoo_autocomplete.png
+For example, after typing 'Odoo', the following information populates:
+
+.. image:: partner_autocomplete/odoo-autocomplete.png
    :align: center
    :alt: Creating a new contact in Odoo
 
-.. image:: partner_autocomplete/odoo_info_autocomplete.png
+In the chatter, the following information populates about the company, after clicking on the desired
+pre-populated contact:
+
+.. image:: partner_autocomplete/odoo-info-autocomplete.png
    :align: center
    :alt: View of the information being shown about odoo with the autocomplete option in Odoo
 
-.. tip:: Partner Autocomplete also works if you enter a VAT number instead of a company name.
+.. tip:: Partner Autocomplete also works if a :abbr:`VAT (value-added tax)` number is entered
+   instead of a company name.
 
 Pricing
 =======
 
-| *Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
-  used. Each request will consume one credit.
-| To buy credits, go to :menuselection:`Settings --> Contacts --> Partner Autocomplete or Odoo IAP -->
-  View My Services` and select a package.
+*Partner Autocomplete* is an *In-App Purchase (IAP)* service, which requires prepaid credits to be
+used. Each request consumes one credit.
+
+To buy credits, go to :menuselection:`Settings app --> Contacts section`. Then, locate either the
+:guilabel:`Partner Autocomplete` feature and click :guilabel:`Buy credits`, or locate the
+:guilabel:`Odoo IAP` feature and click :guilabel:`View My Services`. From the resulting page, select
+a desired package.
 
 .. note::
-   - If you run out of credits, the only information that will be populated when clicking on
-     the suggested company will be the website link and the logo.
-   - If you are on Odoo Online and you have the Enterprise version, you benefit from free trial
-     credits to test the feature.
-   - Learn about our *Privacy Policy* `here <https://iap.odoo.com/privacy#header_2>`_.
+   If the database runs out of credits, the only information populated when clicking on the
+   suggested company will be the website link and the logo.
+
+   Learn about our *Privacy Policy* `here <https://iap.odoo.com/privacy#header_2>`_.
```

Comment:
Please use meangingful link labels rather than "here" or "click here" :)

This first option is preferred as linking to a specific heading _could_ break in the future when the sections are modified.
```suggestion
   Learn about our `Privacy Policy <https://iap.odoo.com/privacy>`_.
```
OR to go to the specific heading (more fragile and not recommended):
```suggestion
   Learn about our `Privacy Policy
   <https://iap.odoo.com/privacy#table_of_content_heading_1710263202838_58>`_.
```

Reply:
Option A